tests/test_physics/test_gaussian/test_symplectics.py

```diff
@@ -172,10 +173,10 @@ def test_MZgate_external_tms(phi_ex, phi_in):
 @given(phi_a=st.floats(0, 2 * np.pi), phi_b=st.floats(0, 2 * np.pi))
 def test_MZgate_internal_tms(phi_a, phi_b):
     """Tests the MZgate is implemented correctly by applying it on one half of a maximally entangled state"""
-    r_choi = np.arcsinh(1.0)
-    # bell = (TMSV(r_choi) & TMSV(r_choi))[0, 2, 1, 3]
+    r_choi = settings.CHOI_R
+    bell = (TMSV(r_choi) & TMSV(r_choi)).get_modes([0, 2, 1, 3])
     MZ = MZgate(phi_a=phi_a, phi_b=phi_b, internal=True)
-    cov = (MZ.bell >> MZ[0, 1]).cov
+    cov = (bell[0, 1, 2, 3] >> MZ[0, 1]).cov
```

Comment:
it's because of the way the bell state is made: after `.get_modes([0,2,1,3])` the mode ordering is `[0,2,1,3]`, so this extra step resets it to `[0,1,2,3]`